src/content/en/fundamentals/accessibility/index.md

```diff
@@ -1,8 +1,355 @@
+  accessibility experiences.
+
+
+Understanding accessibility, its scope, and its impact can make you a better web
+developer. This guide is intended to help you understand how you can make your
+websites accessible and usable for everyone.
+
+"Accessibility" can be difficult to spell, but it doesn't have to be difficult
+to accomplish. In this guide, you will see how to get some easy wins to help
+improve accessibility with minimal effort, how you can use what's built in to
+HTML to create more accessible and robust interfaces, and how to leverage some
+advanced techniques for creating polished accessible experiences.
+
+You'll also find that many of these techniques will help you create interfaces
+that are more pleasant and easy to use for *all* users, not just for those with
+disabilities.
+
+Of course, many developers have only a hazy understanding of what accessibility
+means &mdash; something to do with government contracts, checklists, and screen
+readers, right? &mdash; and there are plenty of misconceptions floating around.
+For example, many developers feel that addressing accessibility will force them
+to choose between creating a delightful and attractive experience, and one that
+is clunky and ugly but accessible.
+
+That is, of course, not the case at all, so let's clear that up before we get
+into anything else. What do we mean by accessibility, and what are we here to
+learn about?
+
+## What is accessibility?
+Broadly speaking, when we say a site is accessible, we mean that the site's
+content is available, and its functionality can be operated, by literally
+*anyone*. As developers, it's easy to assume that all users can see and use a
+keyboard, mouse, or touch screen, and can interact with your page content the
+same way you do. This can lead to an experience that works well for some people,
+but creates issues that range from simple annoyances to show-stoppers for
+others.
+
+Accessibility, then, refers to the experience of users who might be outside the
+narrow range of the "typical" user, who might access or interact with things
+differently than you expect. Specifically, it concerns users who are
+experiencing some type of impairment or disability &mdash; and bear in mind that
+that experience might be non-physical or temporary.
+
+For example, although we tend to center our discussion of accessibility on users
+with physical impairments, we can all relate to the experience of using an
+interface that is not accessible to us for other reasons. Have you ever had a
+problem using a desktop site on a mobile phone, or seen the message "This
+content is not available in your area", or been unable to find a familiar menu
+on a tablet? Those are all accessibility issues.
+
+As you learn more, you'll find that addressing accessibility issues in this
+broader, more general sense almost always improves the user experience for
+everyone. Let's look at an example.
+
+![a form with poor accessibility](imgs/pooraccess.jpg)
```

Comment:
Suggest leaving this as is because it gets difficult to read when floated to the right and scrunched down. Given the context of the article I think it's better to err on the side of legibility.